website/source/api/agent/check.html.md

```diff
@@ -221,7 +224,8 @@ The table below shows this endpoint's support for
   "Shell": "/bin/bash",
   "HTTP": "https://example.com",
   "Method": "POST",
-  "Header": {"x-foo":["bar", "baz"]},
+  "Header": {"Content-Type": "application/json", "x-foo": ["bar", "baz"]},
```

Comment:
Since we have a proper header here, we could remove the foo/bar/baz stuff that isn't particularly realistic.
```suggestion
  "Header": {"Content-Type": "application/json"},
```